helm/karpenter/values.yaml

```diff
@@ -107,9 +107,9 @@ karpenter:
       # -- Repository path to the controller image.
       repository: public.ecr.aws/karpenter/controller
       # -- Tag of the controller image.
-      tag: v0.27.0
+      tag: v0.27.5
```

Comment:
the tag says 0.27.5 but the changelog says 0.28

Reply:
and is there a reason why we do not use retagged imaged like for every other app? :p

Reply:
yes and it is consistent with what AWS does in Karpenter 0.28.0: https://github.com/aws/karpenter/blob/26e2d35069869d1f4c43da7a3d8fcf558f921b4c/charts/karpenter/values.yaml#L102

Reply:
here we are using chart dependencies, I wouldn't know how to combine that with retagged images since we also apply some customizations e.g. [here](https://github.com/giantswarm/karpenter-app/blob/main/helm/karpenter/templates/serviceaccount.yaml)